lib/teleterm/clusters/dbcmd_cli_command_provider_test.go

```diff
@@ -83,49 +86,61 @@ func TestDbcmdCLICommandProviderGetCommand(t *testing.T) {
 					},
 				},
 			}
-			localPort := "1337"
-			gateway := gateway.Gateway{
-				Config: gateway.Config{
+			fakeStorage := fakeStorage{
+				clusters: []*Cluster{&cluster},
+			}
+			dbcmdCLICommandProvider := NewDbcmdCLICommandProvider(fakeStorage, fakeExec{})
+			gateway, err := gateway.New(
+				gateway.Config{
 					TargetURI:             cluster.URI.AppendDB("foo").String(),
 					TargetName:            "foo",
 					TargetSubresourceName: tc.targetSubresourceName,
 					Protocol:              defaults.ProtocolPostgres,
 					LocalAddress:          "localhost",
-					LocalPort:             localPort,
+					WebProxyAddr:          "localhost:1337",
+					Insecure:              true,
+					CertPath:              "../../../fixtures/certs/proxy1.pem",
+					KeyPath:               "../../../fixtures/certs/proxy1-key.pem",
+					CLICommandProvider:    dbcmdCLICommandProvider,
 				},
-			}
-			fakeStorage := fakeStorage{
-				clusters: []*Cluster{&cluster},
-			}
-			dbcmdCLICommandProvider := NewDbcmdCLICommandProvider(fakeStorage, fakeExec{})
+			)
+			require.NoError(t, err)
+			t.Cleanup(func() { gateway.Close() })
```

Comment:
Since `gateway.Config` became a private field and is no longer embedded within `gateway.Gateway`, I had to create a gateway here through `gateway.New`.